scripts/validate-dist/package.json

```
@@ -0,0 +1,11 @@
+{
+  "name": "@guardian/source-validate-dist",
+  "version": "1.0.0",
```

since we won't publish this we should make sure it can't be by accident 😉 

we can probably also do away with the scope/context etc?

```suggestion
  "name": "validate-dist",
  "version": "1.0.0",
  "private": true,
```